src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
@esjewett Yeah, I thought this would prompt some discussion.  I'm not sure exactly how you would like this done to best fit with your particular vision.

offset, one, and zero were currently hidden as private closure variables in the dimension, so I couldn't access them from a higher-level crossfilter method.  They need to be exposed somehow from the dimension itself.  Adding an API or other approach would seem to expose an interface for this internal implementation detail, so that seemed wrong.  Thus the convention of using _ to denote a private member seemed like the easiest alternative.  I'll be happy to change this if you have a cleaner vision how to expose dimension-level state to crossfilter-level methods?